From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Heilig Benedek <benecene@gmail.com>
Date: Thu, 18 Oct 2018 17:08:13 -0700
Subject: color_chooser.patch

Disables a DCHECK that crashes the ColorChooser on Windows,
that DCHECK most likely is an artifact that remained in chromium from a
long time ago (last update of that part of the code was around 2012-2013,
and this is purely UI, I don't think they have automated tests for it).

diff --git a/chrome/browser/ui/views/color_chooser_win.cc b/chrome/browser/ui/views/color_chooser_win.cc
index 434842ba0f81206a43fb26874930bf7309782890..93b4152003eaea05f0e16cd049687fbcbc672fb0 100644
--- a/chrome/browser/ui/views/color_chooser_win.cc
+++ b/chrome/browser/ui/views/color_chooser_win.cc
@@ -91,7 +91,7 @@ void ColorChooserWin::OnColorChooserDialogClosed() {
     color_chooser_dialog_->ListenerDestroyed();
     color_chooser_dialog_ = NULL;
   }
-  DCHECK(current_color_chooser_ == this);
+  // DCHECK(current_color_chooser_ == this);
   current_color_chooser_ = NULL;
   if (web_contents_)
     web_contents_->DidEndColorChooser();
